Documentation/community.rst

```diff
@@ -4,6 +4,67 @@
     Please use the official rendered version released here:
     https://docs.cilium.io
 
+Weekly Community Meeting
+========================
+
+The Cilium contributors gather every Monday at 8am PDT, 17:00 CEST, for a
+one-hour Zoom call open to everyone. During that time, we discuss:
+
+- the statuses of the next releases for each supported Cilium release;
+- the current state of our CI: flakes being investigated and upcoming
+  changes;
+- the development items for the next release;
+- and miscellaneous topics during the open session.
+
+If you want to discuss something during the next meeting's open session,
+you can add it to `the meeting's Google doc
+<https://docs.google.com/document/d/1Y_4chDk4rznD6UgXPlPvn3Dc7l-ZutGajUv1eF0VDwQ/edit#>`_.
+
+Zoom link: https://zoom.us/j/596609673.
+
+Slack
+=====
+
+Our Cilium & eBPF Slack is the main discussion space for the Cilium community.
+Click `here <https://cilium.herokuapp.com>`_ to request an invite. 
+
+Slack channels
+--------------
+
+==================== ====================================
+Name                 Purpose
+==================== ====================================
+#general             General user discussions & questions
+#kubernetes          Kubernetes-specific questions
+#networkpolicy       Questions on network policies
+#release             Release announcements only
+#sig-*               SIG-specific discussions (see below)
```

Comment:
Does this need to be here if it's also listed below for contributors?

Reply:
I wasn't sure if we considered the SIG channels development channels or not. But we actually clarify that in a couple places already and we have #sig-hubble vs. #hubble and #sig-policy vs. #networkpolicy, so SIGs are clearly for development.

I removed this line.